include/rtc/pliresponder.hpp

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2023 Arda Cinar
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef RTC_PLI_RESPONDER_H
+#define RTC_PLI_RESPONDER_H
+
+#if RTC_ENABLE_MEDIA
+
+#include "mediahandlerelement.hpp"
+#include <functional>
+
+namespace rtc {
+
+/// Responds to PLI and FIR messages sent by the receiver. The sender should respond to these
+/// messages by sending an intra. 
+class RTC_CPP_EXPORT PliResponder final : public MediaHandlerElement {
+    std::function<void(void)> onPli;
```

Comment:
`onPli` should be a `Synchronized_callback` for thread safety. Also, private member variables should be prefixed by `m` in the library so the name should be `mOnPli`.